kustomize/commands/edit/add/addreplacement.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package add
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"log"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type addReplacementOptions struct {
+	Replacement types.ReplacementField
+}
+
+func newCmdAddReplacement(fSys filesys.FileSystem) *cobra.Command {
+	var o addReplacementOptions
+	o.Replacement.Source = &types.SourceSelector{}
+	o.Replacement.Targets = append(o.Replacement.Targets, &types.TargetSelector{})
+	cmd := &cobra.Command{
+		Use:   "replacement",
+		Short: "add an item to replacement field",
+		Long: `this command will add an item to  replacement field in the kustomization file.
+the item will be:
+	- be either a file, or an inline string
+`,
+		Example: `add replacement --path {filepath}`,
```

Comment:
suggestion: it'd be nice for UX to expand the example here showing the whole `kustomize edit ...` command, akin to what the examples shown in `kustomize edit add --help` do.

Reply:
this is follow what ``kustomize edit add patch`` done before